content/docs/guides/crosswalk/kubernetes/apps.md

```diff
@@ -51,11 +50,12 @@ Check out how to:
 
   * [Build and Deploy a Container](#build-and-deploy-a-container)
   * [Deploy a Pod with a Sidecar](#deploy-a-pod-with-a-sidecar)
+  * [Deploy a Helm Chart](#deploy-a-helm-chart)
   * [Deploy Wordpress](#deploy-wordpress)
```

Comment:
It looks like you might have removed this section, so this link can go, I think.

Reply:
It's in there :)

Reply:
Ha, you're right. I couldn't find it last night, but that may have been a lack of sleep.